docs/add-ons/presentation.md

```diff
@@ -61,25 +72,108 @@ profile ADDON_SLUG flags=(attach_disconnected,mediate_deleted) {
+Here are the requirements of Ingress:
+- Ingress must be enabled.  Set `ingress: true` in [config.json](/docs/add-ons/configuration#add-on-config).
+- Your server may run on port 8099.  If it does not run on 8099, you must set `ingress_port: PORT_NUMBER` in [config.json](/docs/add-ons/configuration/#add-on-config) to match your configuration.
+- Only connections from `172.30.32.2` must be allowed. You should deny access to all other IP addresses within your add-on server. 
+- Users are previously authenticated via Home Assistant.  Authentication is not required. 
 
-- The add-on will need to provide the web interface on port `8099`. Make sure that the add-on accepts only connections from `172.30.32.2` on that port and that the connections are treated as authenticated.
-- Update add-on configuration and set `ingress: true`. Here it is also possible to configure the Ingress port by using the option `ingress_port: PORT_NUMBER` (default 8099).
-- If you need to configure the application inside your add-on with the right path and port, query the add-on info API endpoint.
-- If the application doesn't support relative paths or you can't set a base url, you can use nginx filter to replace the URL with correct path. Ingress adds a request header `X-Ingress-Path` that can be used.
+:::tip
+Configuration of path and port information may be queried via  [Addons info api endpoint](/docs/api/supervisor/endpoints/#addons). If the Home Assistant url is required by your addon, Ingress adds a request header `X-Ingress-Path` which may be filtered to obtain the base url. 
+:::
 
 Ingress API gateway supports the following:
 
 - HTTP/1.x
 - Streaming content
 - Websockets
+
+
+## Basic Ingress Example with Nginx
+The following is a basic ingress implementation with an Nginx server.  This contains an example`Dockerfile`, `config.json`, and `ingress.conf` configuration.  
+
+The `ingress.conf` is configured to accept only connections from IP address `172.30.32.2` as we are only expecting connections from this IP address for Ingress purposes. Any other IP address will be rejected.  The ingress port 8099 is utilized to reduce configuration work.  If you wish to configure a different ingress port you may, but the `config.json` option `ingress_port` must be defined to match. 
+
+ingress.conf
+```nginx 
+server {
+    listen 8099;
+    allow  172.30.32.2;
+    deny   all;
+}
+```
+
+Our example `Dockerfile` is configured to support only our Nginx server, and does not support a `run.sh` like most addons.  You may replace the `CMD` with your own command to allow more configuration options while launching your addon.   This Dockerfile will `RUN` to install our Nginx dependencies, `COPY` our example ingress.conf from above to the add-on conatiner, then `CMD` start Nginx.  
+
+Dockerfile
+```Dockerfile
+ARG BUILD_FROM
+FROM $BUILD_FROM
+ENV LANG C.UTF-8
+
+#Add nginx and create the run folder for nginx.
+RUN apk add nginx;mkdir -p /run/nginx;
```

Comment:
The default run location differs on Alpine, it is not needed to create that folder.
It is advised to use `--no-cache` in general, to keep the resulting image smaller.

```suggestion
RUN apk --no-cache add nginx
```

Reply:
mkdir -p /run/nginx was required during testing.

Reply:
I've never needed to use it 😄 But 👍